connectors/migrations/20240719_migrate_nango_connection.ts

```diff
@@ -96,18 +98,22 @@ async function migrateConnectionId(
     raw_json: connection.credentials.raw,
   };
 
-  // Below is to be tested with a provider that has refresh tokens
-
-  // if (connection.credentials.expires_at) {
-  //   migratedCredentials.access_token_expiry = Date.parse(
-  //     connection.credentials.expires_at
-  //   );
-  // }
-  // if (connection.credentials.refresh_token) {
-  //   migratedCredentials.refresh_token = connection.credentials.refresh_token;
-  // }
+  // If provider supports refresh tokens, migrate them.
+  if (CONNECTORS_WITH_REFRESH_TOKENS.includes(provider)) {
+    if (connection.credentials.expires_at) {
+      console.log(
+        "connection.credentials.expires_at:",
+        connection.credentials.expires_at
+      );
 
-  // End has to be tested
+      migratedCredentials.access_token_expiry = Date.parse(
+        connection.credentials.expires_at
+      );
+    }
+    if (connection.credentials.refresh_token) {
+      migratedCredentials.refresh_token = connection.credentials.refresh_token;
+    }
+  }
```

Comment:
You want to check that the expiry is at least 30mn in the future before moving forward here

Reply:
Added.